tock/tock/templates/_navigation.html

```diff
@@ -10,11 +10,24 @@
           <span>Projects</span>
         </a>
       </li>
-      <li>
-        <a href="{% url 'reports:ListReports' %}" class="usa-nav-link">
-          <span>Reports</span>
-        </a>
-      </li>
+      {% if user.is_staff %}
+        <li>
```

Comment:
This doesn't render for me.  No menu actually shows up.

Reply:
We may need to be sure you're set up as superuser and staff in the admin.